discovery-provider/src/queries/get_tracks.py

```diff
@@ -75,10 +75,14 @@ def _get_tracks(session, args):
                 "track_id",
             ]
             base_query = parse_sort_param(base_query, Track, whitelist_params)
+    if "slug" in args:
```

Comment:
this fn is a bit odd b/c it's not able to handle single track & multiple tracks. perhaps we should do the same thing we do with "id" as with "slug" and make it an array? wdyt?

or at least move this up to the top and comment that this only fetches a single vs. an array of tracks

Reply:
I think we move it to the top w/ comment. Not sure if there's a use case for getting multiple tracks by slug.